addon/lib/main.js

```diff
@@ -231,11 +248,42 @@ exports.getBackend = function () {
   return backendOverride || prefs.backend;
 };
 
+let infoPanelShownForWindow = null;
+
+function showInfoPanel(magicCookie, title, description) {
+  let win = winutil.getMostRecentBrowserWindow();
+  let target = {
+    node: win.document.getElementById(magicCookie),
+    targetName: magicCookie
+  }
+  UITour.showInfo(win, null, target, title, description);
+  UITour.showHighlight(win, target, "wobble");
+  infoPanelShownForWindow = win;
+}
+
+function hideInfoPanel() {
+  if (infoPanelShownForWindow) {
+    UITour.hideInfo(infoPanelShownForWindow);
+    UITour.hideHighlight(infoPanelShownForWindow);
+    infoPanelShownForWindow = null;
+  }
+}
+
 // For reasons see https://developer.mozilla.org/en-US/Add-ons/SDK/Tutorials/Listening_for_load_and_unload
 exports.main = function (options) {
+  loadReason = options.loadReason;
+  if (options.loadReason === "install") {
+    let helpurl = exports.getBackend() + "/homepage/help.html";
+    let win = winutil.getMostRecentBrowserWindow();
+    win.loadURI(helpurl);
```

Comment:
Yes. Opening a new tab caused issues. I can't remember the exact details at the moment. I should have taken notes.